src/rust/ide/src/view/node_searcher.rs

```diff
@@ -80,6 +86,28 @@ impl NodeSearcher {
         self.display_object.add_child(&self.text_field.display_object());
         self.text_field.clear_content();
         self.text_field.set_focus();
+        let module     = self.node_editor.displayed_module();
+        //TODO[ao]: Now we use some predefined location, until this task will be done:
+        // https://github.com/enso-org/ide/issues/653 . This code should be replaced with
+        // the proper Searcher view integration anyway.
+        let position   = TextLocation { line:2, column:4 };
```

Comment:
Doesn't it bother server if the location is out of bounds?
Perhaps (0,0) would be a safer default?